example_configs/nextcloud_oidc_authelia.md

```diff
@@ -0,0 +1,81 @@
+## Authelia
+
+Set up Authelia according to it's [documentation](https://www.authelia.com/overview/prologue/introduction/), including the [OpenID Connect](https://www.authelia.com/configuration/identity-providers/open-id-connect/) and [Nextcloud instructions](https://www.authelia.com/integration/openid-connect/nextcloud/).
+
+## LLDAP
+
+Set up [LLDAP (Light Lightweight Directory Access Protocol](https://github.com/nitnelave/lldap) according to its standard documentation. 
+
+Add a group and note the name you use. For this tutorial, we're using the group `users`. Create a new user and add it to the `users` group. 
```

Comment:
Maybe we should use `nextcloud_users` here as well? Since we're connecting with nextcloud.

Reply:
Yes, I'm going to chance it across both docs. I'll also check if any other examples reference nextcloud and update there, but let me know if you know of any specifically.